app/src/main/java/mozilla/lockbox/support/TimingSupport.kt

```diff
@@ -56,12 +73,12 @@ open class AutoLockSupport(
         if (autoLockTime == Setting.AutoLockTime.Never) {
             storeAutoLockTimerDate(Long.MAX_VALUE)
         } else {
-            storeAutoLockTimerDate(lockingSupport.systemTimeElapsed + autoLockTime.ms)
+            storeAutoLockTimerDate(systemTimingSupport.systemTimeElapsed + autoLockTime.ms)
         }
     }
 
     private fun lockCurrentlyRequired(): Boolean {
-        val currentSystemTime = lockingSupport.systemTimeElapsed
+        val currentSystemTime = systemTimingSupport.systemTimeElapsed
         return if (currentSystemTime <= Constant.Common.sixtySeconds) {
```

Comment:
I'm not sure I understand why `Constant.Common.sixtySeconds` is not in a property — isn't `autoLockInterval` settable via `SharedPreferences`?